modules/demand_electricity/workflow/Snakefile

```diff
@@ -18,7 +18,10 @@ with open(workflow.source_path("internal/internal_config.yaml"), "r") as f:
 include: "rules/downloads.smk"
 include: "rules/demand.smk"
 
-if config["use_default_customisable_resources"]:
+if config["use_default_user_resources"]:
     rule all:
         message: "Generate all outputs for 'demand_electricity'."
-        input: "results/demand-electricity-ehighways.csv"
+        input:
+            expand("results/demand_electricity_{resolution}.csv", resolution="ehighways"),
+            expand("results/plot_timeseries_{resolution}.png", resolution="ehighways"),
+            expand("results/plot_map_{resolution}.png", resolution="ehighways")
```

Comment:
This is fine for now, but in the future we should move to a small check, so we can use this to generate unit tests.
https://snakemake.readthedocs.io/en/stable/snakefiles/testing.html